web/web.go

```diff
@@ -7,13 +7,14 @@ import (
 	"net/http"
 	"strings"
 
+	"xojoc.pw/useragent"
```

Comment:
@moonmeister Sorry, this repo uses a GPLv3 license which is not compatible with the Mattermost software.

Reply:
Any alternative library we can consider?

Reply:
Oh, Ok. What is compatible?

Reply:
@moonmeister Here's a list that's a good start. https://www.apache.org/legal/resolved.html#category-a

Reply:
According to the Licence it seems only the webapp is under Apache 2.0. the Server seems to be under GNU AGPL v.3.0...Maybe I'm missing something.